[DOC-295] [DOC-432]JumpCloud IdP for YBA (#23214)
yugabyte/yugabyte-db: docs/content/preview/yugabyte-platform/security/authentication/_index.md

    icon="/images/section_icons/secure/authentication.png">}}

  {{<index/item
    title="OIDC authentication"

>     body="Authenticate database users using SSO via Azure AD or JumpCloud as the IdP."

Reviewer: ```suggestion
    body="Use an identity provider to authenticate database users."
```
